README.md

```diff
@@ -28,6 +28,25 @@ Once installed, see [documentation](docs) for example configurations.
 A Docker image is available for containerised environments; see [Docker
 Hub][hub] for more details of the image and available tags.
 
+## Integration snapshots
+
+The `integration` package contains snapshots of data from the integration test
+incident.io account. These snapshots are not intended to be managed by hand, and
+should be updated automatically instead.
+
+Whenever you need to update the snapshots, run:
+
+```console
+$ export TEST_INCIDENT_API_KEY="<test-key>"
+$ export TAP_SNAPSHOT_UPDATE='true' ginkgo -tags=integration -r ./integration
+Writing snapshot file testdata/sync/follow_ups.json
+...
```

Comment:
```suggestion
```

Reply:
I included this as an example of what you're get if you've done it right, would you advise not doing that?

Reply:
fine with that! thought it might be confusing to copy paste it but thats a small nit

Reply:
Ah so with the `console` type I'd normally signify actual commands with a `$` prefix while output lacks it. Helps avoid people copy and pasting a command and accidentally running it with the `$` prefix too!

Reply:
TIL!